exporters/powerdns/exporter.yml

```diff
@@ -14,19 +14,19 @@ exporter_commit: fffbd7c4768681f93988c2c3287b690db20e6ce0
 exporter_changelog: https://github.com/lotusnoir/prometheus-powerdns_exporter/tags
 # Enable packages for Linux
 package_linux: true
-# Enable packages for Windows
-package_windows: false
+# Enable packages for Windowss
+package_windows: true
 # Upgrade GUID used in the msi package. Required if package_windows is set to true
 # This GUID should be generated and be unique across all exporters in the repository
-upgrade_guid: 278db4a4-60a9-493b-b878-dc7b445a2db0
+upgrade_guid: e854d6c0-e0ce-4f9a-b405-d855a3864307
```

Comment:
why changing the GUIDS?

Reply:
Because when we enable `package_windows` in the `exporter.yaml` it [checks](https://github.com/newrelic/newrelic-prometheus-exporters-packages/blob/ef2c56994a97d9145dbb15f9d46f0d5ba0b76646/scripts/common_functions.sh#L129) for GUIDS not being reused. So, I assume we basically copied these guids the first time we created the exporter.yaml files. I didn't change the`ibmmq` and`mongodb` but they are using the same too.

Reply:
oh, then thanks for fixing them :)